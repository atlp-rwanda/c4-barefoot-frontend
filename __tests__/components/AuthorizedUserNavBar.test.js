import React from "react";
import Header from "../../src/components/AuthorizedUserNavBar";
import { mount, shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<AuthorizedUserNavBar />", () => {
  let wrapper;
  it("should be rendered without errors", () => {
    wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
