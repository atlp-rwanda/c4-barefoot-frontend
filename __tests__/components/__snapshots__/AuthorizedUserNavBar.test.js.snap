// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthorizedUserNavBar /> should be rendered without errors 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <Connect(DrawerComponent) />
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-title-3"
        variant="h6"
      >
        <WithStyles(ForwardRef(Link))
          className="makeStyles-link-4"
          color="inherit"
          href="/"
        >
          Barefoot Nomad
        </WithStyles(ForwardRef(Link))>
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Button))
        color="inherit"
        href="/logout"
      >
        Log out
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
